.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,18 @@
 ## Proposed Commit Message
 <!-- Include a proposed commit message because all PRs are squash merged -->
 
+> summary: no more than 72 characters, ideally 50
```

Comment:
"ideally". just drop that . no more than 72.

I guess you picked that because it appends (#XXXX) ?

Reply:
[The git book](https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project) has 50 chars for the summary, but we don't need to be quite so strict; GH ellipsises at 72 characters.

Reply:
Dropped the 50 reference, thanks!